server_action_mass_edit/wizard/mass_editing_wizard.py

```diff
@@ -69,6 +69,52 @@ def default_get(self, fields):
 
         return res
 
+    def onchange(self, values, field_names, fields_spec):
+        first_call = not field_names
+        if first_call:
+            field_names = [fname for fname in values if fname != "id"]
+            missing_names = [fname for fname in fields_spec if fname not in values]
+            defaults = self.default_get(missing_names)
+            for field_name in missing_names:
+                values[field_name] = defaults.get(field_name, False)
+                if field_name in defaults:
+                    field_names.append(field_name)
+
+        server_action_id = self.env.context.get("server_action_id")
+        server_action = self.env["ir.actions.server"].sudo().browse(server_action_id)
+        if not server_action:
+            return super().onchange(values, field_names, fields_spec)
+        dynamic_fields = {}
+
+        for line in server_action.mapped("mass_edit_line_ids"):
+            values["selection__" + line.field_id.name] = "ignore"
+            values[line.field_id.name] = False
+
+            dynamic_fields["selection__" + line.field_id.name] = fields.Selection(
+                [('ignore', _("Don't touch"))], default="ignore"
```

Comment:
This is the only change compared to the code in server_action_mass_edit_onchange. In that version, the selection is empty (`[]`, https://github.com/OCA/server-ux/blob/63bfba0/server_action_mass_edit_onchange/wizard/mass_editing_wizard.py#L48-L49). What is the reason for this change?